intel_extension_for_transformers/llm/operator/csrc/dispatcher/neural_speed.cmake

```diff
@@ -0,0 +1,9 @@
+set(NEURAL_SPEED_URL https://github.com/intel/neural-speed.git)
+set(NEURAL_SPEED_TAG 18720b319d6921c28e59cc9e003e50cee9a85fcc) # kernel-only release v0.2
```

Comment:
@luoyu-intel I think a tag is more reasonable?

Reply:
```suggestion
set(NEURAL_SPEED_TAG bestlav0.1)
```

Reply:
https://github.com/intel/neural-speed/releases/tag/bestlav0.1

Reply:
make sence